app/assets/stylesheets/govuk_publishing_components/components/_big-number.scss

```diff
@@ -1,13 +1,25 @@
 .gem-c-big-number {
   margin-bottom: govuk-spacing(3);
+  font-family: $govuk-font-family;
```

Comment:
I know this has been merged already, but I'd suggest using the [`govuk-typography-common`](https://frontend.design-system.service.gov.uk/sass-api-reference/#govuk-typography-common) mixin here instead – which will take care of things like:

- anti-aliasing
- swapping out the font when printing
- any other changes to the 'core' typography introduced in the future